test/racingcar/car_test.go

```diff
@@ -0,0 +1,32 @@
+package racingcar_test
+
+import (
+	"testing"
+
+	"github.com/poi1649/go-racing-car/racingcar"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewCarNameValidation(t *testing.T) {
+	testCases := []struct {
+		name string
+	}{
+		{""},
+		{"123456"},
+		{"abcdef"},
+		{"한글다섯글자"},
```

Comment:
P5: 저희 팀의 경우 가급적 필드명은 생략하지 않는 편이에요. (모든 팀의 컨벤션인지는 모르겠습니다)

```suggestion
		{name: ""},
		{name: "123456"},
		{name: "abcdef"},
		{name: "한글다섯글자"},
```

Reply:
헉 넵!! 감사합니다!
가독성적인 측면 생각해서 필드가 하나더라도 생략 안하는게 좋기도하고, 다른부분들이랑 통일성도 좋겠네요